src/support/socket.h

```diff
@@ -118,7 +118,7 @@ struct SockAddr {
     std::string host = url.substr(2, sep - 3);
     std::string port = url.substr(sep + 1, url.length() - 1);
     ICHECK(ValidateIP(host)) << "Url address is not valid " << url;
-    if (host == "localhost") {
+    if (host == "127.0.0.1") {
```

Comment:
This should not be necessary, since it's already that value :-)

Reply:
This should have been left as it was with "localhost" since it is a special valid address.